### PR TITLE
Feature: Disable EnableBlockServiceFallbackToArchiver by default.

### PR DESCRIPTION
**Pull request by gmalouf**
## Summary

Given there are no archival server records on mainnet/testnet, repeatedly doing queries for them (and adding warn messages to logs) makes little sense at the moment.

## Test Plan
Existing tests pass.

### CODE REVIEW

File: network/wsNetwork_test.go [hunk 1]
```diff
@@ -4136,82 +4136,95 @@ func TestRefreshRelayArchivePhonebookAddresses(t *testing.T) {
...
-		}
+			assert.ElementsMatch(t, primaryRelayResolvedRecords, relayAddrs)
 
-		assert.ElementsMatch(t, primaryRelayResolvedRecords, relayAddrs)
+			archivePeers = netA.GetPeers(PeersPhonebookArchivers)
 
-		archivePeers = netA.GetPeers(PeersPhonebookArchivers)
+			if refreshTestConf.EnableBlockServiceFallbackToArchiver {
+				// For the time being, we do not dedup resolved archive nodes
+				assert.Equal(t, len(primaryArchiveResolvedRecords)+len(secondaryArchiveResolvedRecords), len(archivePeers))
```
Comment (winder) [inline network/wsNetwork_test.go:4206]: This isn't a change is it, just removing the magic number?

File: network/wsNetwork_test.go [hunk 2]
```diff
@@ -4136,82 +4136,95 @@ func TestRefreshRelayArchivePhonebookAddresses(t *testing.T) {
...
-	})
+		})
+	}
+
+	testRefreshWithConfig(defaultConfig)
+
+	configWithBlockServiceFallbackToArchiverEnabled := config.GetDefaultLocal()
+	configWithBlockServiceFallbackToArchiverEnabled.EnableBlockServiceFallbackToArchiver = true
+
+	testRefreshWithConfig(configWithBlockServiceFallbackToArchiverEnabled)
```
Comment (winder) [inline network/wsNetwork_test.go:4227]: :+1:

**Review by winder — DISMISSED** [review]
LGTM

File: network/wsNetwork_test.go (hunk 1 above)
Comment (gmalouf) [reply network/wsNetwork_test.go:4206]: I saw an opportunity to fix a magic number that probably should not have been there.

**Review by onetechnical — APPROVED** [review]

**Review by algonautshant — APPROVED** [review]
Looks good.
You can also remove this:
catchup/universalFetcher_test.go
97:     blockServiceConfig.EnableBlockServiceFallbackToArchiver = false

**Review by algonautshant — APPROVED** [review]

**Pull request merged**